lib/stack_master/stack.rb

```diff
@@ -85,10 +92,17 @@ def self.generate(stack_definition, config)
           tags: stack_definition.tags,
           parameters: parameters,
           template_body: template_body,
+          template_format: template_format,
           notification_arns: stack_definition.notification_arns,
           stack_policy_body: stack_policy_body)
     end
 
+    def self.identify_template_format(template_body)
+      return :json if template_body =~ /^{/x # ignore leading whitespaces
```

Comment:
do we have access to the file extension here? it might be more straight forward using that if we can.